### PR TITLE
fix(autonat): export `DEFAULT_PROTOCOL_NAME`

### PR DESCRIPTION
**Pull request by dignifiedquire**
# Description

This is useful when checking for identified connections and their support for their support of `autonat`.

## Change checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: protocols/autonat/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# Unreleased
```
Comment (mxinden) [inline protocols/autonat/CHANGELOG.md]:
```suggestion
# 0.4.1
```

**Review by mxinden — COMMENTED** [review]
Thanks @dignifiedquire! If you bump the patch version in `protocols/autonat/Cargo.toml` I can do a patch release right away.

**Comment by dignifiedquire** [comment]
done

**Review by mxinden — APPROVED** [review]

**Comment by mxinden** [comment]
[Tagged](https://github.com/libp2p/rust-libp2p/releases/tag/libp2p-autonat-0.4.1) and [published](https://crates.io/crates/libp2p-autonat/0.4.1).

**Pull request merged**